rsopt/libe_tools/libensemble-rsmpi.py

```diff
@@ -2,16 +2,17 @@
 import argparse
 import subprocess
 import sys
+import time
 
-run_string = 'rsmpi -n {n} -h {h} {args}'
+run_string = 'rsmpi -n {n} -h {h} -t {t} {args}'
 
 parser = argparse.ArgumentParser(description="Wrap rsmpi for use by libensemble. Converts required MPICH flags"
                                              "to rsmpi flags.")
 
 parser.add_argument("-np", dest="n", required=True,
                     help="Pass number of processors to rsmpi")
-parser.add_argument("--ppn", dest="ppn", required=False,
-                    help="Not required by rsmpi, but supplied by libEnsemble. Input is not used if given.")
+parser.add_argument("--ppn", dest="t", required=False,
```

Comment:
Is there any reason to have this default to a value? It is currently not `required` and will probably not act as expected if it is omitted

Reply:
Good point. [8c93416](https://github.com/radiasoft/rsopt/pull/125/commits/8c93416d7121c4a6cb56f58f19f1450b147bfaba)